deploy/k8s/deploy.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
+# shellcheck disable=SC1091
```

Comment:
yes, I tried this already.

Reply:
and what was the result of this try? Did it work? If not, then why it didn't?

Reply:
Getting this error 
```
source "${K8S_DIR}"/central.sh
       ^-- [SC1091](https://www.shellcheck.net/wiki/SC1091) (info): Not following: deploy/k8s/central.sh was not specified as input (see shellcheck -x).
````

Checked with this error and its best to ignore these errors. It seems that since this K8S_DIR is dynamic link, hence shellcheck is not able to locate this file.

Reply:
This location is placed in a variable to properly discover the path to that file when we execute it. The location is not dynamic for the linting purposes - it is known where the files are located.

As these files contain export and source directives, I see little sense of calling shellcheck on them and then disabling the directives inside of these files.

There are 2 options we can do here:

## Disable checking of single files

```
# shellcheck source=/dev/null
source "${K8S_DIR}/central.sh"
# shellcheck source=/dev/null
source "${K8S_DIR}/sensor.sh"
```
this would allow to run shellchek without any params. Or

## Allow shellcheck to follow the files

```
# shellcheck source=deploy/k8s/central.sh
source "${K8S_DIR}/central.sh"
# shellcheck source=deploy/k8s/sensor.sh
source "${K8S_DIR}/sensor.sh"
```

and run shellcheck with `-x` parameter (to allow following the sources).